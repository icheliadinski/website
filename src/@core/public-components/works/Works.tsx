import React from 'react'
import { SideCard } from '@core/ui-kit'

export const Works = () => {
  return (
    <SideCard>
      <article>My works</article>
    </SideCard>
  )
}
