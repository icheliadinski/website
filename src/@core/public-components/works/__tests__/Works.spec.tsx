import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Works } from '../Works'

describe('[Works]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Works />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
