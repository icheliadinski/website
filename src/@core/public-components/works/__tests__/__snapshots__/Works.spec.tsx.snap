// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Works] renders successfully 1`] = `
<SideCard>
  <article>
    My works
  </article>
</SideCard>
`;
